src/Sulu/Bundle/PageBundle/Tests/Functional/SmartContent/QueryBuilderTest.php

```diff
@@ -463,15 +463,15 @@ public function testSecurityWithoutPermissionCheck()
         $request = new Request([], [], ['_sulu' => new RequestAttributes(['webspace' => $webspace])]);
         $this->getContainer()->get('request_stack')->push($request);
 
-        $root = $this->sessionManager->getContentNode('test_io');
```

Comment:
@alex `test_io` has `<security permission-check="true">` as the name is **WITHOUT**PermissionCheck i have changed this to `sulu_io` which has `<security permission-check="false">`

Reply:
Make sense. Thank you for explanation!